network/src/peers/peer/peer.rs

```diff
@@ -175,6 +190,78 @@ impl Peer {
     }
 
     pub(super) fn set_disconnected(&mut self) {
+        self.sync_state.reset();
         self.quality.disconnected();
     }
+
+    pub fn start_rtt_measurement(&mut self) {
```

Comment:
```suggestion
    pub fn start_ping_measurement(&mut self) {
```
since we calculate multiple RTTs